### DISCUSSION TITLE
Add Support for Dotfiles

### DISCUSSION THREAD

**Discussion by oliverlabs** [Azure/CloudShell#324] [Ideas]
Why not bring support for dotfiles for Azure CloudShell similar to GitHub Codespaces?

**Comment by theJasonHelmick** [comment]
Thanks @oliverlabs for the suggestion. Can you describe the scenario or problem your trying to solve that requires dot files in Cloud Shell?

**Reply by oliverlabs** [reply]
I don't have any problems and I don't specifically use Cloud Shell on a daily basis, but I thought it'd be cool to have my custom aliases and plugins, and, perhaps, zsh shell with oh-my-zsh plugin when I _do_ use it.

**Comment by theJasonHelmick** [comment]
Thanks for the suggestion @oliverlabs!
